### PR TITLE
i18n(es): Translated `alpinejs.mdx` 

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `alpinejs.mdx` to spanish  🚀

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/alpinejs.mdx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+import Video from '~/components/Video.astro';
+
+Esta **[integración de Astro][astro-integration]** agrega [Alpine.js](https://alpinejs.dev/) a tu proyecto para que puedas usar Alpine.js en cualquier lugar de tu página.
+
+## Instalación
+
+### Instalación rápida
+
+La herramienta de línea de comandos `astro add` automatiza la instalación para ti. Ejecuta uno de los siguientes comandos en una nueva ventana de terminal. (Si no estás seguro de qué gestor de paquetes estás usando, ejecuta el primer comando.) Luego, sigue las instrucciones y escribe "y" en la terminal (es decir, "sí") para cada uno.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/alpinejs.mdx:19]:
```suggestion
La herramienta de línea de comandos `astro add` automatiza la instalación por ti. Ejecuta uno de los siguientes comandos en una nueva ventana de terminal. (Si no estás seguro de qué gestor de paquetes estás usando, ejecuta el primer comando.) Luego, sigue las instrucciones y escribe "y" en la terminal (es decir, "sí") para cada uno.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR, te dejo un par de sugerencias

File: src/content/docs/es/guides/integrations-guide/alpinejs.mdx [hunk 2]
```diff
@@ -0,0 +1,109 @@
+---
+type: integration
+title: '@astrojs/alpinejs'
+description: Learn how to use the @astrojs/alpinejs framework integration to extend component support in your Astro project.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/integrations-guide/alpinejs.mdx:4]:
```suggestion
description: Aprenda a usar la integración de @astrojs/alpinejs para ampliar la compatibilidad de Alpinejs en tu proyecto de Astro.
```

File: src/content/docs/es/guides/integrations-guide/alpinejs.mdx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+### Instalación manual
+
+Primero, instala el paquete `@astrojs/alpinejs` usando tu gestor de paquetes. Si estás usando npm o no estás seguro, ejecuta esto en la terminal:
+
+```sh
+npm install @astrojs/alpinejs
+```
+
+La mayoría de los gestores de paquetes instalarán las dependencias asociadas como pares también. Sin embargo, si ves un aviso "No se puede encontrar el paquete 'alpinejs'" (o similar) al iniciar Astro, deberás instalar manualmente Alpine.js:
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/integrations-guide/alpinejs.mdx:40]:
```suggestion
La mayoría de los gestores de paquetes también instalarán las dependencias asociadas. Sin embargo, si ves un aviso "No se puede encontrar el paquete 'alpinejs'" (o similar) al iniciar Astro, deberás instalar manualmente Alpine.js:
```

File: src/content/docs/es/guides/integrations-guide/alpinejs.mdx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+## Solución de problemas
+
+Para obtener ayuda, consulte el canal `#support` en [Discord](https://astro.build/chat). ¡Nuestros amables miembros del Equipo de Soporte están aquí para ayudar!
+
+También puedes consultar nuestra [documentación de integraciones de Astro][astro-integration] para obtener más información sobre las integraciones.
+
+## Contribuyendo
+
+Este paquete es mantenido por el equipo central de Astro. ¡Estás invitado a enviar un problema o PR!
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/integrations-guide/alpinejs.mdx:101]:
```suggestion
Este paquete es mantenido por el equipo de Astro. ¡Estás invitado a abrir una issue o PR!
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Gracias @Waxer59 y @dreyfus92!

**Pull request merged**